openstack/cronus/templates/maillog/secrets.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.maillog.enabled -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: maillog
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: maillog
+    chart: maillog
+    release: maillog
+    heritage: {{ .Release.Service }}
+type: Opaque
+data:
+  config.yaml: {{ include "cronus/templates/maillog/_config.yaml.tpl" . | b64enc }}
```

Comment:
Ive always this confusing in the other charts.... that we take the config.yaml.tpl, then stich it in the config.yaml of a secret. Why not just use the secret right away? Or stick with the configmap? I.E. why a config that goes to a secret then back to a config.... you could just take the config map and mount it

Reply:
Will resolve this conversation at it is not that important really. Ideally IMHO config files would come from configmaps, but it really does not matter that much